### ISSUE TITLE
Bonusprogram for cleaning

### ISSUE THREAD

**Issue by derguy** [thankthemaker/sharespresso#22]
When a user cleans the coffee-machine he/she will get some coffee for free.

**Comment by thankthemaker** [comment]
Hi Guy,
isn't this a feature request for the ionic app?

**Comment by derguy** [comment]
Yes. I moved the issues from the ionic app to sharespresso to get a better overview of the open issues. Most of them affect both, sharespresso and the ionic app. 

Is it possible to get an information if the coffeemachine is in cleaning-mode? If so then it would be nice when the "cleaner" could identify himself to the system while cleaning and the bonus/credit would be automatically applied.
